style.css

```diff
@@ -113,7 +113,282 @@ html {
   color: rgba(80, 95, 121, 1);
 }
 
-.social-content {
+.works {
+  background-color: #f7f7f9;
+  margin: 0;
+  display: grid;
+  grid-template-rows: auto;
+  padding-top: 80px;
+}
+
+.work-flex {
+  height: auto;
+  background: #ebebff;
+  margin: 0 15px 80px 15px;
+  border-style: solid;
+  border-color: rgb(219, 214, 214);
+  border-width: 1px;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] All works look like cards, with a white background and a specific color for the border. You should consider changing your styling to match the original design better, you can check it out here:

![image](https://user-images.githubusercontent.com/66967274/143578813-63535d3f-e107-4d6c-9c88-15f71e1542ed.png)

Your design:
![image](https://user-images.githubusercontent.com/66967274/143579141-1d008291-bc22-4284-aa83-7b98d27eda3b.png)